php/libraries/Utility.class.inc

```diff
@@ -739,6 +739,35 @@ class Utility
         return $instruments;
     }
 
+    /**
+     * Get a list of instruments installed in Loris.
```

Comment:
description does not match what function is doing, it matches what the previous function is doing !